backend/capellacollab/cli/ws.py

```diff
@@ -0,0 +1,355 @@
+# SPDX-FileCopyrightText: Copyright DB Netz AG and the capella-collab-manager contributors
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import contextlib
+import select
+import sys
+import time
+import uuid
+from pathlib import Path
+from typing import Annotated
+
+import typer
+from kubernetes import client, config, stream
+from websocket import ABNF  # type: ignore[import]
+
+app = typer.Typer()
+
+MOUNT_PATH = "/workspace"
+
+
+@app.callback()
+def init_kube():
+    config.load_kube_config()
+
+
+def get_current_namespace():
+    try:
+        _, active_context = config.list_kube_config_contexts()
+        return active_context["context"]["namespace"]
+    except KeyError:
+        return "default"
+
+
+NamespaceOption = typer.Option(
+    "--namespace",
+    "-n",
+    help="Kubernetes namespace",
+    default_factory=get_current_namespace,
+)
+
+
+@app.command()
+def volumes(namespace: Annotated[str, NamespaceOption]):
+    """List all Persistent Volume Claims in a kubernetes namespace."""
+    core_api = client.CoreV1Api()
```

Comment:
I wanted to avoid global state. What I actually wanted is an injected dependency, such as we have in FastAPI. However, I did not investigate that yet.

Anyway, I would keep the commands as isolated as possible.